content/docs/command-reference/remote/modify.md

```diff
@@ -262,6 +262,22 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify --local myremote sse_kms_key_id 'key-alias'
   ```
 
+- `sse_customer_key` - key to encrypt data uploaded when using
+  [SSE-C](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ServerSideEncryptionCustomerKeys.html).
+  The value should be a base64-encoded 256 bit key.
+
+  ```dvc
+  $ dvc remote modify --local myremote sse_customer_key 'mysecret'
+  ```
+
+- `sse_customer_algorithm` - server-side encryption algorithm to use with
+  `sse_customer_key`. This parameter will be passed directly to AWS S3, so DVC
```

Comment:
Does this NOT apply to `sse_kms_key_id` then?

Reply:
As pointed in the comment above, `sse_kms_key_id` will be relevant only if users want to use keys from AWS KMS. For SSE-C, since the user will be specifying their own key, `sse_kms_key_id` will be ignored.